src/main/scala/org/ergoplatform/local/MempoolAuditor.scala

```diff
@@ -71,19 +72,21 @@ class MempoolAuditor(nodeViewHolderRef: ActorRef,
       nodeViewHolderRef ! GetNodeViewChanges(history = false, state = true, mempool = true, vault = false)
```

Comment:
Seems like the message `log.warn(s"Cleanup worker failed during initialization with: $e")`  is not correct

Reply:
You comment has no connection with the line of the code.

    case e: ActorInitializationException =>
      log.warn(s"Cleanup worker failed during initialization with: $e")
      Stop

is correct